### PR TITLE
Add two missing properties to React.Widgets

### PR DESCRIPTION
**Pull request by olmobrutall**
Just adding three missing properties: 

* DropdownList defaultOpen (x2): Is not in the examples but is referenced in `open` documentation.
* DateTimePicker defaultCurrentDate: Is not in the examples but is consistent with the other Uncontrollable properties. Of course works and is quite usefull.

### CODE REVIEW

File: react-widgets/lib/DropdownList.d.ts [hunk 1]
```diff
@@ -87,10 +87,14 @@ interface DropdownListProps extends React.Props<DropdownListClass> {
     onSearch?: (searchTerm: string) => void;
     /**
      * Whether or not the DropdownList is open. When unset (undefined) the DropdownList will
-     * handle the opening and closing internally. The defaultOpen prop can be used to set an
-     * initialization value for uncontrolled widgets.
+     * handle the opening and closing internally. 
      */
     open?: boolean;
+	/**
```
Comment (DanielRosenwasser) [inline react-widgets/lib/DropdownList.d.ts]: Switch to spaces.

File: react-widgets/lib/DateTimePicker.d.ts [hunk 2]
```diff
@@ -31,6 +31,10 @@ interface DateTimePickerProps extends React.Props<DateTimePickerClass> {
      * @default Date()
      */
     currentDate?: Date;
+	/*
```
Comment (DanielRosenwasser) [inline react-widgets/lib/DateTimePicker.d.ts]: Switch to spaces.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
Think you could take a quick look @rogierschouten?

**Comment by olmobrutall** [comment]
Fixed

**Review by DanielRosenwasser — APPROVED** [review]

File: react-widgets/lib/DateTimePicker.d.ts [hunk 3]
```diff
@@ -31,6 +31,10 @@ interface DateTimePickerProps extends React.Props<DateTimePickerClass> {
      * @default Date()
      */
     currentDate?: Date;
+    /*
+     * Default value for current date. Usefull for suggesting a date when the caldenar opens without keep forcing it once 'value' is set.
```
Comment (DanielRosenwasser) [inline react-widgets/lib/DateTimePicker.d.ts]: Nit: Useful.

**Comment by olmobrutall** [comment]
Fixed

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**